.github/workflows/leeyabot.yml

```diff
@@ -1,9 +1,7 @@
 name: leeyabot
 on:
   pull_request:
-    branches:
-      - master
-      - v3_dev
+    branches: [main, master, v3_dev]
```

Comment:
I think we can remove the v3_dev from here right? also we don't have a main so can delete that as well I think